tools/run_tests/run_performance_tests.py

```diff
@@ -462,7 +462,23 @@ def run_collect_perf_profile_jobs(hosts_and_base_names, scenario_name):
                   const=True,
                   help='Run netperf benchmark as one of the scenarios.')
 argp.add_argument('--perf_args',
-                  help='Wrap QPS workers in a perf command, with arguments to perf specified here.')
+                  help=""" 
+                          Example usage: "--perf_args=record -F 99 -g".
```

Comment:
yeah it did turn out that indentation was included in the result, updated without using """